examples/intkey_go/src/sawtooth_intkey_client/utils_test.go

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
The commit message makes it sound like it a unit test for the intkey client but this is not the case, also listing "1." for only one item is redundant. I suggesting changing the commit message to "Add unit test for SHA512 conversion method in Go" also rename the file to something that does not mention utils.

Reply:
This commit became obsolete with refactored commit in place, file name is now main_test.go
Let me squash it with previous commit as there's only one unit test case here.